mage_integrations/mage_integrations/transformers/base.py

```diff
@@ -0,0 +1,177 @@
+from datetime import datetime
+from mage_ai.data_preparation.models.pipeline import Pipeline
```

Comment:
there could be circular dependencies if you reference mage_ai library here since mage_ai library depends on mage_integrations as well.